examples/tof-viewer/src/ADIMainWindow.cpp

```diff
@@ -2598,10 +2595,3 @@ int ADIMainWindow::saveIniFile() {
 
     return 0;
 }
-
-void ADIMainWindow::iniParamWarn(std::string variable, std::string validVal) {
```

Comment:
We already fixed this and merged it to main. Please see: https://github.com/analogdevicesinc/ToF/pull/603